bundles/org.openhab.binding.verisure/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,747 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="verisure"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="bridge">
```

Comment:
No, it is not a public API, I have reverse engineered it. But we can call it verisureApi.

Reply:
After thinking about it, it would then break all current users using things-file where they have used brdige type id `bridge`. I think I will keep the name, even if it is quite common.